From e90c02932912f326280343a7d66fa5a545f08392 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 2 Jan 2024 11:35:59 +0100
Subject: [PATCH 0287/1695] dt-bindings: soc: add some RIF-aware peripheral IDs

Add some RIF-aware peripheral IDs to "stm32mp25-rifsc.h"

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 25014c2e1e1a3d4b3f57a25db453449831b198ba dt-bindings: soc: add some RIF-aware peripheral IDs]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I836a5ed237dfaba23ed86df2fc68f84d2c894ee9
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/324733
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/include/dt-bindings/soc/stm32mp25-rifsc.h | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/core/include/dt-bindings/soc/stm32mp25-rifsc.h b/core/include/dt-bindings/soc/stm32mp25-rifsc.h
index 036bb5811..18c23ec94 100644
--- a/core/include/dt-bindings/soc/stm32mp25-rifsc.h
+++ b/core/include/dt-bindings/soc/stm32mp25-rifsc.h
@@ -124,6 +124,21 @@
 #define STM32MP25_RIFSC_OTFDEC2_ID		126
 #define STM32MP25_RIFSC_IAC_ID			127
 
+/* RIF-aware IPs */
+#define STM32MP25_RIFSC_PWR_ID			155
+#define STM32MP25_RIFSC_GPIOA_ID		160
+#define STM32MP25_RIFSC_GPIOB_ID		161
+#define STM32MP25_RIFSC_GPIOC_ID		162
+#define STM32MP25_RIFSC_GPIOD_ID		163
+#define STM32MP25_RIFSC_GPIOE_ID		164
+#define STM32MP25_RIFSC_GPIOF_ID		165
+#define STM32MP25_RIFSC_GPIOG_ID		166
+#define STM32MP25_RIFSC_GPIOH_ID		167
+#define STM32MP25_RIFSC_GPIOI_ID		168
+#define STM32MP25_RIFSC_GPIOJ_ID		169
+#define STM32MP25_RIFSC_GPIOK_ID		170
+#define STM32MP25_RIFSC_GPIOZ_ID		171
+
 #define RIFSC_RISC_CFEN_SHIFT			0
 #define RIFSC_RISC_SEM_EN_SHIFT			1
 #define RIFSC_RISC_SCID_SHIFT			4
-- 
2.47.2

